packages/cactus-plugin-satp-hermes/src/main/typescript/blo/admin/get-healthcheck-handler-service.ts

```diff
@@ -0,0 +1,41 @@
+import { GetStatusError } from "../../core/errors/satp-errors";
+import { HealthCheckResponse } from "../../generated/gateway-client/typescript-axios/api";
+import { Logger } from "@hyperledger/cactus-common";
+import { SATPManager } from "../../gol/satp-manager";
+
+export async function ExecuteGetHealthCheck(
```

Comment:
@AndreAugusto11 Please use `camelCase` for function names. Applies to all functions throughout the PR/codebase